### DISCUSSION TITLE
The DIFF window in GitHub Copilot Chat is really small. Can it be enlarge?

### DISCUSSION THREAD

**Discussion by doggy8088** [community/community#127849] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

The DIFF window in GitHub Copilot Chat is really small. Can it be enlarge?

![image](https://github.com/community/community/assets/88981/f3c45179-f1b7-4d4f-af8d-7077ea503ba4)

**Comment by Akash1134** [comment, marked as answer]
Hi @doggy8088 ,

Thanks for sharing your valuable feedback. One of our engineers have already responded to [your issue](https://github.com/microsoft/vscode-copilot-release/issues/1286) in the Vscode-copilot-release repo. Please follow up with the team on that issue.
